src/components/clusterConfiguration/NetworkConfigurationFormFields.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Form } from 'formik';
```

Comment:
I think you want import 'Form' from patternfly instead of formik. That will fix the styling because PF Form component has pf-c-form class which adds spacing between form groups.

Reply:
Thank you!